lib/pure/parsecsv.nim

```diff
@@ -29,8 +32,8 @@
 ## reference for item access with `rowEntry <#rowEntry.CsvParser.string>`_:
 ##
 ## .. code-block:: nim
-##   import parsecsv
-##   import os
+##   import os, parsecsv
```

Comment:
`import os` is not needed for this example

Reply:
> import os is not needed for this example

Thx. I did it.